assets/scss/components/_header.scss

```diff
@@ -159,3 +159,39 @@ a.header__menu__proposition-name {
     text-decoration: none;
   }
 }
+
+
+.header__menu__language-selection-links {
```

Comment:
We already have styles for [language select](https://github.com/hmrc/assets-frontend/blob/master/assets/components/header/_header.scss#L51-L61).

How does the markup for these proposed styles differ from the [existing markup](https://github.com/hmrc/assets-frontend/blob/master/assets/components/header/header.html#L47-L54) and why?

Reply:
I'll try to get the functionality split tomorrow for you as I'm due to go into meetings, but also need to speak to a colleague.

I'll also expand on the reasons for the new link styling and why the existing lang-select will not work in this case.

Kind regards,